src/Network/Network/help/New-AzApplicationGatewayIdentity.md

```diff
@@ -0,0 +1,109 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azapplicationgatewayidentity
+schema: 2.0.0
+---
+
+# New-AzApplicationGatewayIdentity
+
+## SYNOPSIS
+Creates an identity object for an application gateway. This will hold reference to the user assigned identity.
+
+## SYNTAX
+
+```
+New-AzApplicationGatewayIdentity -UserAssignedIdentityId <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+**New-AzApplicationGatewayIdentity** cmdlet creates an application gateway identity object.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\>$identity = New-AzUserAssignedIdentity -Name $identityName -ResourceGroupName $rgName -Location $location
+PS C:\>$appgwIdentity = New-AzApplicationGatewayIdentity -UserAssignedIdentity $identity.Id
```

Comment:
@akshaysngupta this example should be updated to use the result of this cmdlet in `New-AzApplicationGateway` to show the end-to-end flow